Android/app/build.gradle

```diff
@@ -15,6 +15,18 @@ try {
     keystoreProperties['storePassword'] = ''
 }
 
+// Go backend build constants
+def goToolChain = 'go1.21.1'
+def goSourceDir = "${projectDir}/src/main/go"
+def goSourcePackages = ["${goSourceDir}/intra",
+                        "${goSourceDir}/intra/android",
+                        "${goSourceDir}/intra/doh",
+                        "${goSourceDir}/intra/split",
+                        "${goSourceDir}/intra/protect"]
+def goBuildDir = file("${buildDir}/go")
+def goBackendAAR = file("${goBuildDir}/tun2socks.aar")
```

Comment:
Rename to go.aar?

tun2socks is no longer an appropriate name

Reply:
What about `backend.aar` to be more specific to the feature instead of the language.